app/uploaders/image_uploader.rb

```diff
@@ -31,6 +33,10 @@ def store_dir
     process resize_to_fill: [50, 50]
   end
 
+  version :thumb_no_crop, if: :is_comment? do
```

Comment:
If we want to make this more 4chan-esque, I do think bounding it by [125,125] is more suitable. Even if we're not going for that specifically, [250,250] is a bit too big.